gitgud/levels/__init__.py

```diff
@@ -2,20 +2,21 @@
 from gitgud.levels.merging import all_challenges as merging_challenges
```

Comment:
You can change this line to `from gitgud.levels.merging import level as merging_level`. You'll also need to change `all_challenges` to `level` for this line to work. Do the same thing for the merging level